src/lib/mina_base/parties.ml

```diff
@@ -376,35 +376,32 @@ module Transaction_commitment = struct
 
   let typ = Snark_params.Tick.Field.typ
 
-  let create ~other_parties_hash ~protocol_state_predicate_hash ~memo_hash : t =
+  let create ~other_parties_hash : t =
     Random_oracle.hash ~init:Hash_prefix.party_with_protocol_state_predicate
-      [| protocol_state_predicate_hash; other_parties_hash; memo_hash |]
+      [| other_parties_hash |]
```

Comment:
We're double-hashing here, I feel like we could use `other_parties_hash` directly

Reply:
@mrmr1993 do not need to apply the hash prefix for this?

Reply:
It should be fine to use it directly, since it's already a hash